tests/tests.rs

```diff
@@ -578,6 +578,34 @@ fn no_system_header_includes() {
         .success());
 }
 
+// TODO write a test that makes sure multiple include dirs with relative prefixes handled correctly
```

Comment:
I'm not sure how multiple include dirs show up in the paths that the callbacks receive -- any suggestions for how to exercise weird scenarios? I think that they're all supposed to be printed relative to CWD.

Reply:
> Is it something like -I foo and then a file doing #include "bar.h", and making sure that the deps file has foo/bar.h? Or something else?

Yes, this is one case I'm thinking of. The other that comes to mind is when a system header gets included, like if I pick up a header from Xcode's global include directory, does the path get encoded correctly?

Reply:
OK, adding to the buildscript for the integration crate covered the main case I'm thinking of. I think that for system headers and the like I'll see if any issues arise in practice, can always add more tests if I need to fix bugs when actually carrying the integration through.